### ISSUE TITLE
Is netwulf available in anaconda?

### ISSUE THREAD

**Issue by zhaohan-xi** [benmaier/netwulf#40]
I use conda virtual environment, normally install a new library by "conda install -n my_env_name package_name", but netwulf is unavailable now, is there any surrogate approaches to install netwulf into my conda virtual environment?

**Comment by ulfaslak** [comment]
No. But could you not install pip within your environment using conda, and then install netwulf with that version of pip?

**Comment by benmaier** [comment]
I have never worked with conda but this seems related: https://stackoverflow.com/questions/29286624/how-to-install-pypi-packages-using-anaconda-conda-command

In case somebody wants to register a Conda recipe, we're happy to merge pull requests. Please regard our [contributing guidelines](https://github.com/benmaier/netwulf/blob/master/CONTRIBUTING.md).
